runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -333,11 +367,23 @@ impl<T: Config> Pallet<T> {
 			// Note that `provide_multi_dispute_data` will iterate, verify, and import each
 			// dispute; so the input here must be reasonably bounded.
 			let _ = T::DisputesHandler::provide_multi_dispute_data(disputes.clone())?;
+			dispute_sets_processed_metric
+				.with_label_values(&vec!["imported)"])
```

Comment:
is closing brace `)` intended here?

Reply:
No, thanks for spotting this typo/copy paster error.